python/ray/train/tests/util.py

```diff
@@ -22,3 +23,11 @@ def load_dict_checkpoint(checkpoint: Checkpoint) -> Dict[str, Any]:
     with checkpoint.as_directory() as checkpoint_dir:
         with open(os.path.join(checkpoint_dir, "data.pkl"), "rb") as f:
             return ray_pickle.load(f)
+
+
+def mock_storage_context() -> StorageContext:
+    return StorageContext(
+        storage_path=tempfile.mkdtemp(),
```

Comment:
Will this get cleaned up eventually when the test finishes?

Reply:
No, we need this thing for tests involving `TuneController`